presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -122,6 +122,7 @@
     EXCEEDED_REVOCABLE_MEMORY_LIMIT(0x0002_000B, INSUFFICIENT_RESOURCES),
     EXCEEDED_LOCAL_BROADCAST_JOIN_MEMORY_LIMIT(0x0002_000C, INSUFFICIENT_RESOURCES),
     EXCEEDED_OUTPUT_POSITIONS_LIMIT(0x0002_000D, INSUFFICIENT_RESOURCES),
+    LOCAL_TASK_ERROR(0x0002_000E, INTERNAL_ERROR, true),
```

Comment:
- Let's remove `true`. It is not used for this scenario. 
- Let's move this to INTERNAL_ERROR section. Otherwise, the error code would be wrong....